sources/SilkTouch/SilkTouch/Mods/MixKhronosData.cs

```
@@ -1813,33 +1997,171 @@ private class Rewriter(JobData job) : CSharpSyntaxRewriter(true)
             }
 
             var nativeName = node.AttributeLists.GetNativeTypeName();
-            if (nativeName is null || !nativeName.StartsWith("#define "))
+            switch (nativeName)
```

This particular bit of code should probably be a common function

---

I decided to replace `ModUtils.GetNativeElementTypeName` with `TryParseNativeTypeName`, which handles both the original functionality of parsing inputs that look like `const NAME **` and the new functionality of parsing inputs that look like `#define NAME VALUE`.

The method's implementation is not exhaustive and will silently drop any unsupported formats, such as `const uint32_t[8]`. I considered logging or throwing an exception, but logging is noisy and exceptions don't work well for intentionally unimplemented cases that are commonly hit.

My intention with this method is that we implement cases as we need it and consolidate any NativeTypeName parsing to here.

---

Sounds good.